webpack/assets/javascripts/react_app/components/AutoCompleteSearch/helpers.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Menu, MenuItem } from 'react-bootstrap-typeahead';
+import groupBy from 'lodash/groupBy';
+import URI from 'urijs';
+import UUID from 'uuid/v1';
+import SubstringWrapper from '../common/SubstringWrapper';
+import { TRIGGERS } from './AutoCompleteConstants';
+
+export const renderMenu = ({ results, searchQuery }) => {
```

Comment:
Why is it an `helper` and not a `component`?

Reply:
Eventually this is a prop for `Autocomplete` 
though I can see why it can become a component.